### ISSUE TITLE
save interval by internal from #1

### ISSUE THREAD

**Issue by kevincobain2000** [kevincobain2000/email_extractor#3]
creating separate issue from #1

@sirolele 

> Also if possible to save interval by internal not until it crawl all urls Eg. if we are to crawl 40,000 pages website. it will crawl all before it grep one email to file

**Comment by sirolele** [comment]
One last question. So untill it finishes before it will save to output to some.txt

**Comment by kevincobain2000** [comment]
yes after it is implemented. at the present moment it crawls each url first and then it saves to file

**Comment by sirolele** [comment]
OKay but this worse than the first script. 
For Example like i have a web with over 40K Pages. I must finishe crawling all before it saves.
There is a limit which the unix memory hold internal before saving, i found out i only had 40 emails after leaving it for long hours. so i am back to formal script with allow me to set no of urls to crawl

**Comment by sirolele** [comment]
can you edit the new one to allow me set no of url and it will be 100% perfect. so i can leave it an run and sleep

**Comment by kevincobain2000** [comment]
@sirolele 
You mean the number of urls (uri's) to crawl under a set domain?
I think there already is an option in the script `-l 10`

with limit 2

```
07/25/15 @12:28PM - kevincobain2000@Mac Air ~/Downloads/email_extractor@master* python email_extractor.py -u http://computer-fixperts.com/ -l 2
Namespace(file=None, limit=2, silent=False, url='http://computer-fixperts.com/')
  http://computer-fixperts.com/
  http://computer-fixperts.com
enquiries@computerfixperts.com.au
```

with limit 5

```
07/25/15 @12:29PM - kevincobain2000@Mac Air ~/Downloads/email_extractor@master* python email_extractor.py -u http://computer-fixperts.com/ -l 5
Namespace(file=None, limit=5, silent=False, url='http://computer-fixperts.com/')
  http://computer-fixperts.com/
  http://computer-fixperts.com
  http://computer-fixperts.com/about-us/
  http://computer-fixperts.com/services/
  http://computer-fixperts.com/locations/
enquiries@computerfixperts.com.au
```

default is unlimited
